app/src/main/java/org/oppia/app/home/topiclist/PromotedStoryListViewModel.kt

```diff
@@ -15,6 +15,6 @@ class PromotedStoryListViewModel(private val activity: AppCompatActivity) : Home
   }
 
   override fun routeToContinuePlaying() {
-    activity.startActivity(ContinuePlayingActivity.createContinuePlayingActivityIntent(activity.applicationContext))
+    activity.startActivity(RecentlyPlayedActivity.createContinuePlayingActivityIntent(activity.applicationContext))
```

Comment:
createContinuePlayingActivityIntent to createRecentlyPlayedActivityIntent.

Reply:
Done